lib/stagehand/staging/synchronizer.rb

```diff
@@ -7,7 +7,7 @@ module Synchronizer
       BATCH_SIZE = 1000
       SESSION_BATCH_SIZE = 30
       ENTRY_SYNC_ORDER = [:delete, :update, :insert].freeze
-      ENTRY_SYNC_ORDER_SQL = ActiveRecord::Base.send(:sanitize_sql_for_order, ['FIELD(operation, ?), id DESC', ENTRY_SYNC_ORDER]).freeze
+      ENTRY_SYNC_ORDER_SQL = ActiveRecord::Base.send(:sanitize_sql_for_order, [Arel.sql('FIELD(operation, ?), id DESC'), ENTRY_SYNC_ORDER]).freeze
```

Comment:
You'll need to also wrap the entire thing or it'll throw up the warning when you use this constant.
```suggestion
      ENTRY_SYNC_ORDER_SQL = Arel.sql(ActiveRecord::Base.send(:sanitize_sql_for_order, [Arel.sql('FIELD(operation, ?), id DESC'), ENTRY_SYNC_ORDER])).freeze
```

Also weird that they make no mention of this in their documentation. They literally have this exact example and don't say anything about needing to Arel it. https://api.rubyonrails.org/classes/ActiveRecord/Sanitization/ClassMethods.html#method-i-sanitize_sql_for_order